### ISSUE TITLE
[css color-mix] Feature request: Physical color mixing option (Kubelka-Munk)

### ISSUE THREAD

**Issue by NateBaldwinDesign** [w3c/csswg-drafts#8715]
Regarding https://www.w3.org/TR/css-color-5/#funcdef-color-mix

## Overview
Incorporate a *physical color mixing* option (eg Kubelka–Munk) alongside the current interpolation options for the css `color-mix()` function.

## Why am I making this request?
Interpolations in perceptual color spaces are great and have many uses; however the title of the spec feature ("mix") implies a far different behavior of color. Interpolating at a certain percentage from one color to another is not the same as "mixing" colors in the physical world. While I can appreciate it's a more human-friendly term than "color-interpolate()" it almost begs for better methods of returning a "mixed" color.

Many people are unfamiliar with the behavior of light, light mixing, perceptual space, and color interpolation. Supporting a function that returns colors that closely resemble their physically mixed counterparts may do well for meeting the expectations of certain people. At the very least, it would be a good addition to the options, considering each color space/method returns different results, and the decision on which to use can be highly subjective.

I would love to see the addition of a physical color mixing approach, such as [Kubelka–Munk](https://en.wikipedia.org/wiki/Kubelka%E2%80%93Munk_theory), which more closely reflects the physical result of mixing colors. Examples of this method in implementation can be found here: https://github.com/scrtwpns/mixbox and here: https://github.com/rvanwijnen/spectral.js

## Related images
Here's an example of interpolation between `#002286` and `#fdd200` in a variety of color spaces:
<img width="596" alt="image" src="https://user-images.githubusercontent.com/13972198/231981069-e1443f62-db91-4f9d-a294-35d898c3baa9.png">

And another of OKLCh
<img width="959" alt="image" src="https://user-images.githubusercontent.com/13972198/231981661-c7da1bc8-991b-4f82-b5bf-e88699a842db.png">

For comparison against a Kubelka-Munk mixing of the two colors, we can see that the green is much deeper and richer (two examples):
<img width="806" alt="image" src="https://user-images.githubusercontent.com/13972198/231982724-1761e0a5-873f-47dd-a987-d21180685818.png">

<img width="827" alt="image" src="https://user-images.githubusercontent.com/13972198/231982381-a19ca050-b80c-4c75-b48d-caf60acac4c0.png">

**Comment by CaliOn2** [comment]
Can I get an update on where this is at right now?

**Comment by rvanwijnen** [comment]
Hi, I'm the creator of Spectral.js, an MIT-licensed Kubelka-Munk JavaScript library.
I'd be happy to help make this a reality.

https://github.com/rvanwijnen/spectral.js

<img width="2356" height="1753" alt="Image" src="https://github.com/user-attachments/assets/4057ffcd-ed3c-4070-ad75-4528f32ab21b" />

**Comment by svgeesus** [comment]
The thing is that there are so many simulations of physical pigment mixing, for various types of paints and inks.K-M is just one, albeit  a popular one.

**Comment by rvanwijnen** [comment]
That’s true—there are many different simulations for physical pigment mixing, and Kubelka-Munk is just one of them. But it doesn’t always have to be an academic or purely scientific approach. In practice, tools like MixBox and Spectral.js use simpler models that still produce results most people find very satisfying.

**Comment by MarkJeronimus** [comment]
> And another of OKLCh

Watch out with interpolating polar coordinates. Because the hue of black, gray and white is not defined, it often defaults to 0, which is reddish:

<img width="594" height="118" alt="Image" src="https://github.com/user-attachments/assets/c48144ca-d316-4ac2-b8d0-0c254e545f8f" />

**Comment by svgeesus** [comment]
> Because the hue of black, gray and white is not defined, it often defaults to 0, which is reddish:

I agree some poor implementations do this. 

This is why, in CSS, [we explicitly cater for powerless or missing color coordinates](https://drafts.csswg.org/css-color-4/#missing) especially when interpolating colors or creating relative colors.
